README.md

```diff
@@ -53,3 +53,11 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the tests
+To run the test for codebase run the following command:
+
+Poetry run pytest
```

Comment:
Minor, but notice that the capital P in poetry here will work in some environments (Windows mostly!) but not others where lowercase/uppercase commands are not treated as identical!

`poetry run pytest`